frontend/src/components/changeRole.js

```diff
@@ -0,0 +1,17 @@
+// Change Role dropdown
+// handler function must be included for logic
+import React from 'react'
+import { permissionRolesEnum } from '../utils/enums'
+
+const ChangeRole = ({ memberID, role, handleChange }) => (
```

Comment:
any reason why we can't include the handler function here?

Reply:
i think it's very possible for us to reuse this component in the future and we can't guarantee that it'll have the same handler function